src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -231,6 +231,14 @@ export class OrganizeImportsAction extends EditorAction {
 				primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KeyO,
 				weight: KeybindingWeight.EditorContrib
 			},
+			metadata: {
+				description: "Organize imports by removing unused imports, sorting imports and combining imports from the same module",
```

Comment:
Try seeing if you can only update the description to include optimize imports. Also keep the description generic as this command is used for many different languages. For example, eome of these do removed unused imports , while others do not